src/main/java/org/springframework/data/couchbase/repository/query/StringBasedN1qlQueryParser.java

```diff
@@ -192,11 +191,11 @@ private PlaceholderType checkPlaceholders(String statement) {
 		}
 
 		if (posCount > 0) {
-			return PlaceholderType.POSITIONAL;
+			return placeHolderType = PlaceholderType.POSITIONAL;
```

Comment:
assignment and return in one ?

Reply:
Yes. It used to be called int the constructor as 
     this.placeHolderType = checkPlaceholders(...)
Now it is public and gets called by StringN1qlQueryCreator - but the placeHolderType still needs to be set. I suppose it could be changed to not return a value - it was private, nothing else could be calling it for that.

I'll change it.